nemo/collections/nlp/models/language_modeling/megatron_gpt_prompt_learning_model.py

```diff
@@ -506,6 +530,12 @@ def on_test_epoch_end(self):
         logging.info(f'test_loss: {averaged_loss[0]}')
         self.test_step_outputs.clear()  # free memory
 
+    def setup(self, stage=None):
+        super().setup(stage)
+
+        if self.frozen_model.cfg.get('transformer_engine', False):
+            self.frozen_model.setup_transformer_engine_tp_groups()
```

Comment:
I think we need to update this for mcore_gpt. We need to setup those groups when we are using it.

Reply:
you mean
```
if self.cfg.get('transformer_engine', False) or self.cfg.get('mcore_gpt', False):
            self.setup_transformer_engine_tp_groups()
```

Reply:
done.